vdyp-sindex/src/main/Sindxdll.java

```diff
@@ -0,0 +1,3748 @@
+ *               bhage=0 ht=1.3 to bhage=0.5 ht=1.3.
+ *               Added "AC" to the end of the define.
+ *               They are now the default for their species:
+ *               ACT_THROWERAC, BA_KURUCZ82AC, BL_CHENAC, BP_CURTISAC,
+ *               HM_MEANSAC, FDI_THROWERAC, ACB_HUANGAC, PW_CURTISAC,
+ *               HWC_WILEYAC, FDC_BRUCEAC, CWC_KURUCZAC, PY_HANNAC,
+ *               SE_CHENAC.
+ *      jul 28 - Updated citation of Nigh's 1999 Pl and 2003 Fdc.
+ *      aug 7  - v1.32 Added 40 more species.
+ *      sep 11 - v1.33 Added Fd, Pl, Hw, Cw.
+ *      dec 15 - v1.34 Added more coast/interior designations for dozens of codes.
+ *          16 - v1.35 Changed Pw location designation to none.
+ * 2004 feb 10 - v1.36 Changed Cw,Fd,Hw from 0 to 5 in spec_use.
+ *             - Changed Pl from 0 to 6 in spec_use.
+ *      mar 26 - v1.37 Added SW_GOUDIE_NATAC.
+ *      apr 28 - v1.38 Added Nigh's 2002 Py.
+ *      may 4  - Updated citation and notes for Nigh's Py.
+ *          5  - Changed default Py to Nigh's.
+ *             - Changed default Sw natural curve to be SW_GOUDIE_NATAC.
+ *      jun 15 - v1.39 Added Nigh's 2004 Pl/Sw/Se total age curves, but not available
+ *               on their own.
+ *             - Substituted Nigh's total age curves for the 0-1.3m area of
+ *               Pl Thrower, Sw Goudie Nat & Pla AC, and Se Chen AC.
+ *             - Changed default Sw curve from SW_GOUDNIGH to SW_GOUDIE_PLAAC.
+ *             - Changed default Pli curve from PLI_THROWNIGH to PLI_THROWER.
+ *      jul 12 - v1.40 Added Pli Thrower to Age-to-Age function.
+ *             - Added check for age-to-age conversion going negative.
+ *      sep 14 - v1.41 Several si2ht.c changes of checking BHage.
+ * 2005 feb 18 - v1.42 Bug fix in Fdc Bruce and Bruce AC, to use total age
+ *               directly instead of converting to breast-height age.
+ *      oct 20 - Added jack pine (PJ).
+ * 2008 feb 28 - Added 2004 Sw Nigh GI.
+ *      jul 4  - v1.43 Release.
+ * 2009 may 6  - v1.44 Force pure y2bh for Fdc-Bruce.
+ * 2009 aug 28 - Added Nigh's 2009 Ep.
+ * 2010 mar 4  - v1.45 Added Nigh's 2009 Ba GI.
+ *             - Added Nigh's 2009 Ba, as default.
+ *      apr 14 - Added 2010 Sw Hu and Garcia.
+ * 2014 apr 25 - v1.47 Added Sindex_AgeSIToHtSmooth().
+ *      sep 2  - Added 2014 Se Nigh GI.
+ * 2015 apr 9  - v1.48 Removed SI_SPEC_BV.
+ *          23 - Updated three "in review"/"in press" citations for Py, Pl, Sw, At.
+ *             - Updated text notes of SI_PY_NIGH.
+ *      may 13 - v1.49 Added 2015 Se Nigh.
+ * 2016 mar 9  - v1.50 Added Sindex_Y2BH05() which os now the rounded value,
+ *               changing Sindex_Y2BH() to be the unrounded value.
+ * 2017 feb 2  - v1.51 Added Nigh's 2016 Cwc equation as default.
+ * 
+ * 2023 jun 1 - Translated from C to Java
+ */
+
+
+//Taken from sindxdll.h (I have commented out the unused ones and labeled them as such)
+/** 

```

Comment:
Comments applying to multiple following constructs or where the thing they refer to is commented out should not be javadoc comments, otherwise javadoc parsers will get confused about what the comment applies to.  When I brought up javadoc before I really meant only the comments explaining `public` members of the class should be converted to Javadoc.  

For instance, javadoc will think this comment refers to `SI_ESTAB_NAT` as it is the next member of the class after this. comment.